CefSharp.OffScreen/ChromiumWebBrowser.cs

```diff
@@ -614,20 +678,77 @@ void IRenderWebBrowser.InvokeRenderAsync(BitmapInfo bitmapInfo)
         /// in it's lock scope.
         /// </summary>
         /// <param name="bitmapInfo">information about the bitmap to be rendered</param>
+        /// <exception cref="InvalidOperationException">When background image is null when on blend mode.</exception>
         public virtual void InvokeRenderAsync(BitmapInfo bitmapInfo)
         {
             var gdiBitmapInfo = (GdiBitmapInfo)bitmapInfo;
-            if (bitmapInfo.CreateNewBitmap)
+
+            if (BlendingConfiguration == PopupBlending.Separate)
             {
-                if (Bitmap != null)
+                if (bitmapInfo.CreateNewBitmap)
                 {
-                    Bitmap.Dispose();
-                    Bitmap = null;
+                    if (Bitmap != null)
+                    {
+                        Bitmap.Dispose();
+                        Bitmap = null;
+                    }
+
+                    Bitmap = gdiBitmapInfo.CreateBitmap();
                 }
+            }
+            else if (BlendingConfiguration == PopupBlending.Blend)
```

Comment:
`Background` is not really a great name here, just `bitmap` and `popup` would be fine. For simplicity, it's just a matter of storing `popup`, then writing `popup` onto `bitmap` when a screen shot is requested.